swssconfig/swssconfig.cpp

```diff
@@ -39,9 +43,28 @@ void dump_db_item(KeyOpFieldsValuesTuple &db_item)
     SWSS_LOG_DEBUG("]");
 }
 
+ProducerStateTable &get_producer_table(const std::string &table_name)
```

Comment:
>get_producer_table [](http://example.com/codeflow?start=20&length=18)

This is a common pattern "find and insert". You may achieve it by one lookup. [https://stackoverflow.com/a/1409465/2514803](https://stackoverflow.com/a/1409465/2514803)

Reply:
sure, I will fix